src/site/_data/contributors.js

```diff
@@ -1222,6 +1222,15 @@ const contributors = {
       en: 'Software Engineer on the web platform.',
     },
   },
+  megginkearney: {
+    name: {
+      given: 'Meggin',
+      family: 'Kearney',
+    },
+    description: {
+      en: 'Meggin is a Tech Writer.',
+    },
+  },
```

Comment:
I think there was recently a different PR that added Meggin to `contributors.js`. Please check if this is a duplicate now.